apps/web/modules/ui/components/icons/reddit-icon.tsx

```
@@ -0,0 +1,31 @@
+export const RedditIcon: React.FC<React.SVGProps<SVGSVGElement>> = () => {
+  return (
+    <svg width="17" height="16" viewBox="0 0 17 16" fill="none" xmlns="http://www.w3.org/2000/svg">
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)